filebeat/docs/reference/configuration/filebeat-options.asciidoc

```diff
@@ -386,27 +405,14 @@ If the specified path is not absolute, it is considered relative to the configur
 filebeat.config_dir: path/to/configs
 -------------------------------------------------------------------------------------
 
-===== encoding
-
-The file encoding to use for reading files that contain international characters.
-See the encoding names http://www.w3.org/TR/encoding/[recommended by the W3C for use in HTML5].
-
-Here are some sample encodings from W3C recommendation:
-
-    * plain, latin1, utf-8, utf-16be-bom, utf-16be, utf-16le, big5, gb18030, gbk, hz-gb-2312,
-    * euc-kr, euc-jp, iso-2022-jp, shift-jis, and so on
+include::../../../../libbeat/docs/shipperconfig.asciidoc[]
```

Comment:
We should probably rename this file from shipper config, as shipper part does not exist anymore. This is mainly for the url ...